.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[isort]
+line_length=120
+multi_line_output=0
+lines_after_imports=2
```

Comment:
not 100% sure about this. when the first thing after an import is a function/class we already have two empty lines inbetween, but without this setting it's only one empty line for other things like variables.